cyipopt/scipy_interface.py

```diff
@@ -377,8 +404,9 @@ def minimize_ipopt(fun,
         Extra keyword arguments passed to the objective function and its
         derivatives (``fun``, ``jac``, ``hess``).
     method : str, optional
-        This parameter is ignored. `minimize_ipopt` always uses Ipopt; use
-        :py:func:`scipy.optimize.minimize` directly for other methods.
+        If unspecified (default), Ipopt is used.
+        :py:func:`scipy.optimize.minimize` methods can also be used as long
+        as no `kwargs` are specified.
```

Comment:
```suggestion
        :py:func:`scipy.optimize.minimize` methods can also be used.
```
(Originally, this ability was excluded from the scope, but I added it.)